src/neptune/new/cli/__init__.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+__all__ = [
+    "AbstractBackendRunner",
```

Comment:
Does all these things should be exposes outside of `cli` submodule? I'm not sure but I expect to only expose `status` and `sync` methods.

Reply:
If this was just copy-pasted let's think of getting some minimal subset that should be user-exposed :wink: